app/views/redirections/_redirection.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr>
+  <%= content_tag(:td, redirection.slug) %>
```

Comment:
> Do you like the other way better or are you just curious?

When I see `content_tag`, I assume something fancy is happening. In this case, nothing fancy is happening, so it confused me for a sec.